plugins/org.jkiss.dbeaver.ext.ocient/src/org/jkiss/dbeaver/ext/ocient/model/OcientMetaModel.java

```diff
@@ -0,0 +1,120 @@
+import org.jkiss.dbeaver.ext.generic.model.GenericTableBase;
+import org.jkiss.dbeaver.ext.generic.model.GenericStructContainer;
+import org.jkiss.dbeaver.ext.generic.model.GenericView;
+import org.jkiss.dbeaver.ext.generic.model.meta.GenericMetaModel;
+import org.jkiss.dbeaver.model.DBUtils;
+import org.jkiss.dbeaver.model.exec.jdbc.JDBCPreparedStatement;
+import org.jkiss.dbeaver.model.exec.jdbc.JDBCResultSet;
+import org.jkiss.dbeaver.model.exec.jdbc.JDBCSession;
+import org.jkiss.dbeaver.model.runtime.DBRProgressMonitor;
+
+import java.sql.SQLException;
+import java.util.Map;
+
+public class OcientMetaModel extends GenericMetaModel {
+
+    public OcientMetaModel() {
+        super();
+    }
+    
+    @Override
+    public GenericTableBase createTableOrViewImpl(
+        GenericStructContainer container,
+        @Nullable String tableOrViewName,
+        @Nullable String tableOrViewType,
+        @Nullable JDBCResultSet dbResult)
+    {
+        if (isView(tableOrViewType))
+        {
+            return new OcientView(
+                container,
+                tableOrViewName,
+                tableOrViewType,
+                dbResult);
+        } else {
+            return new OcientTable(
+                container,
+                tableOrViewName,
+                tableOrViewType,
+                dbResult);
+        }
+    }
+
+    protected String getObjectDDL(DBRProgressMonitor monitor,
+            GenericTableBase sourceObject, Map<String, Object> options,
+            String sql, String description) throws DBException {
+        GenericDataSource dataSource = sourceObject.getDataSource();
+
+        try (JDBCSession session = DBUtils.openMetaSession(monitor, sourceObject, description)) {
+            try (JDBCPreparedStatement stmt = session.prepareStatement(sql)) {
+                try (JDBCResultSet resultSet = stmt.executeQuery()) {
+                    StringBuilder createText = new StringBuilder();
+                    while (resultSet.nextRow()) {
+                        String creationDDL = resultSet.getString(1);
+
+                        if (creationDDL != null)
+                            createText.append(creationDDL);
+                    }
+
+                    if (createText.length() == 0) {
+                        return "-- Create text not found";
+                    }
+                    else {
+                        return createText.toString();
+                    }
+                }
+            }
+        }
+        catch (SQLException e) {
+            throw new DBException(e, dataSource);
+        }
+    }
+
+    public String getViewDDL(DBRProgressMonitor monitor,
+            GenericView sourceObject, Map<String, Object> options) throws DBException {
+    	String schemaName = sourceObject.getContainer().getName();
+        String viewName = sourceObject.getName();
+        String viewDDLSQL = String.format("export view %s.%s", schemaName, viewName);
```

Comment:
It is better to use DBUtils.getFullQualifiedName instead of `%s.%s`  concatenation.
getFullQualifiedName quotes schema and table names if needed (in case of special characters in identifiers).